src/test/functional/steps/view-money-claims-result.ts

```diff
@@ -0,0 +1,39 @@
+import { Then, When } from 'cucumber';
+import * as I from '../utlis/puppeteer.util';
+import { expect } from 'chai';
+
+When('I select {string} category from the list of categories', async (option: string) => {
+  const element = option;
+  await I.click(element);
+});
+
+Then('I select an {string} of either send documents, get an application update or not listed', async (option: string) => {
+  let element;
+  switch(option) {
+    case 'document court': {
```

Comment:
i think its better to pass in the element as an feature example than switch cases. not a big deal tbh.

Reply:
Changed to passing in element as example.